sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1023,6 +1061,7 @@ export class BlockBlobClient extends BlobClient {
     constructor(url: string, pipeline: Pipeline);
     commitBlockList(blocks: string[], options?: BlockBlobCommitBlockListOptions): Promise<BlockBlobCommitBlockListResponse>;
     getBlockList(listType: BlockListType, options?: BlockBlobGetBlockListOptions): Promise<BlockBlobGetBlockListResponse>;
+    query(query: string, options?: BlockBlobQueryOptions): Promise<BlobDownloadResponseModel>;
```

Comment:
should `query()` returns `BlobQueryResponse`?

Reply:
We can make`BlobQueryResponse` alias of `BlobDownloadResponseModel`

Reply:
oh so the type in `sdk/storage/storage-blob/src/BlobQueryResponse.ts` is only for internal use?

Reply:
`BlobQueryResponse` is implementation of `BlobDownloadResponseModel` interface

Reply:
Got it.